include/libswiftnav/time.h

```diff
@@ -17,6 +17,34 @@
 
 #include <libswiftnav/common.h>
 
+/** Number of days in a year. */
+#define YEAR_DAYS           365
+#define LEAP_YEAR_DAYS      (YEAR_DAYS + 1)
+
+/** Number of days in a week. */
+#define WEEK_DAYS           7
+
+/** Number of months in a year. */
+#define YEAR_MONTHS         12
+
+/** Days in (leap) year 1980 since GPS epoch Jan 6th */
+#define YEAR_1980_GPS_DAYS  361
+
+/** Year of GPS epoch */
+#define GPS_EPOCH_YEAR      1980
+
+/** UTC (SU) offset */
+#define UTC_SU_OFFSET       3
+
+/** Number of seconds in a minute. */
+#define MINUTE_SECS         60
+
+/** Number of minutes in an hour. */
+#define HOUR_MINUTES        60
+
+/** Number of seconds in an hour. */
+#define HOUR_SECS           (MINUTE_SECS * HOUR_MINUTES)
+
 /** Number of seconds in a week. */
 #define WEEK_SECS (7*24*60*60)
```

Comment:
Perhaps we can also add an DAY_HOURS? then we could make this WEEK_DAYS * DAY_HOURS * HOUR_SECS